### PR TITLE
feat: sorting

### LINKED ISSUE #4: US: Sorting
No need for subtasks in this one since it is only one functionality we want to add:

Ability to change the sorting order by clicking column headers. Whichever column is pressed latest will be the one that we fort first on. Pressing the header again should toggle ascending/descending. Same as the old table.

This should be done with a patch back-end call (soft properties). In sense the property panel should handle that and display the discard/confirm dialog when going back to edit mode. Outside sense the changes are simply discarded after the session ends.

### PR DESCRIPTION
**Pull request by haxxmaxx**
fixes #4

There is weird lag for column styling that is dependent on the cell value. The values are updated before the the styling is. We should consider if it is good enough before starting to look at it IMO.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -37,8 +39,8 @@ export default function supernova(env) {
 
       const [pageInfo, setPageInfo] = useState({ top: 0, height: 100 });
       const [muiParameters] = useState(muiSetup(__OPIONAL_THEME_DEPS__));
-
       const [tableData] = usePromise(() => manageData(model, layout, pageInfo), [layout, pageInfo]);
+      const changeSortOrder = useMemo(() => sortingFactory(model, layout), [layout]);
```
Comment (Caele) [inline src/index.js:43]: Unsure about using useMemo here, think you could just as well call it directly.

**Review by Caele — APPROVED** [review]
Looks good

File: src/index.js (hunk 1 above)
Comment (Caele) [reply src/index.js:43]: Also, layout could come from the TableHeader, its only model that needs to be hoisted to the function.

Comment (haxxmaxx) [reply src/index.js:43]: Trying to avoid fetching if I don't need to, but it probably don't make any difference

**Pull request merged**